azure-pipelines.yml

```diff
@@ -9,14 +9,28 @@ trigger:
 jobs:
 - job: WinBuild
   displayName: Windows Build
-  pool:
-    vmImage: 'vs2017-win2016'
   strategy:
     matrix:
-      Debug:
-        CI_ENV_BUILD_TYPE: Debug
-      Release with Release Installer:
-        CI_ENV_BUILD_TYPE: Release
+#        windows-2022-Debug:
```

Comment:
Do we really need debug builds? This would only be useful in the case that matches all of the following:

 - There is an issue that we can't reproduce.
 - The issue requires attaching a debugger as opposed to just printing more things to the logs.
 - There is an user that can use a debugger.
 - The user can't build Barrier themselves.

I think the likelihood for such problems is very small and worst case we can build barrier ourselves.

Reply:
Sure, I was merely mirroring your feedback in the Mac PR, and I didn't see any harm in including it here. Would you like me to remove the Debug builds?

Reply:
OK. So, my proposal is to remove the Debug builds, keep the Release builders, and generate installers from the 2016 agent, if that one works. I will use a condition for the installer part, based on the imageName variable - not the `CI_ENV_BUILD_TYPE`, which means less complexity. Thoughts?

Reply:
Sounds good.

Reply:
Resolved in be2f658f.